api/src/controllers/website.js

```diff
@@ -40,7 +40,7 @@ async function addWebPage(req, res, next) {
   try {
     const foundWebsiteInfo = await Website.findOne({ teamId });
     if (foundWebsiteInfo) {
-      foundWebsiteInfo.pages.push(req.body.pageName);
+      foundWebsiteInfo.pages = foundWebsiteInfo.pages.concat(req.body.pageName);
```

Comment:
does the former not work?

Reply:
I should've renamed `pageName` to `pageArray` so that it's less confusing, but the reason why push doesn't work is because `pageName` became an array instead of just the page name. So it would be pushing an array into an array rather than a string into the array. I'll rename the variables accordingly.